index.bs

```diff
@@ -197,14 +201,13 @@ after step 6, add the following step:
 Standard metadata names {#standard-metadata-names}
 ------------
 For the section <a href="https://html.spec.whatwg.org/multipage/#standard-metadata-names">standard metadata names</a>,
-add a subsection named `accept-ch` with the [=Accept-CH state|outlined explanation=].
+add a subsection named `accept-ch` with the [=Accept-CH state|outlined explanation=]. Note that this should be run exactly once,
+and only on an element in the root Document which has not been modified by any script execution.
```

Comment:
This does not seem interoperably implementable. When exactly should it be run? What if there are multiple elements with accept-ch? What if there are zero; how are we supposed to run this exactly once? (Plus the problem mentioned above about "modified by script execution" not being a defined term.)